test/unit/test_compiler_client.py

```diff
@@ -66,16 +73,14 @@ def test_sets_timeout_on_requests(mocker: MockerFixture):
         assert client.timeout == compiler_client.timeout
 
 
-def test_get_version__returns_version(mocker: MockerFixture):
+def test_get_version__returns_version(monkeypatch: MonkeyPatch):
     client_configuration = QCSClientConfiguration.load()
     compiler_client = CompilerClient(client_configuration=client_configuration)
 
-    rpcq_client = patch_rpcq_client(mocker=mocker, return_value={"quilc": "1.2.3"})
+    version_mock = AsyncMock(return_value="1.2.3")
+    monkeypatch.setattr(qcs_sdk, "get_quilc_version", version_mock)
 
     assert compiler_client.get_version() == "1.2.3"
-    rpcq_client.call.assert_called_once_with(
-        "get_version_info"
-    )
 
```

Comment:
Why was `MockerFixture` replaced with `MonkeyPatch`? Does the former not support async?

Reply:
Noting that this removes the `assert_called_once` check

Reply:
At one point I was having issues with `MockerFixture`, which motivated the change to `MonkeyPatch`. This `AsyncMock` solution I landed on though works in combination with `MockerFixture` just fine, so I swapped it and put the deleted assert back in.